dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -177,3 +177,11 @@ cookies.samesite.unsupportedWildcard.useraction=Ensure that only a single wildca
 cookies.samesite.knownDuplicateName=CWWKT0037W: A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations.
 cookies.samesite.knownDuplicateName.explanation=The specified value was already found by the configuration and mapped to a SameSite value.
 cookies.samesite.knownDuplicateName.useraction=Ensure that cookie names and patterns are unique across the lax, none, and strict configuration lists.
+
+trusted.header.origin.invalid.value=CWWKT0038W: The configured trustedHeaderOrigin value [{0}] is invalid and will not be used.
+trusted.header.origin.invalid.value.explanation=The specified value is not a full IP address or correctly formatted hostname.
+trusted.header.origin.invalid.value.useraction=Ensure that configured IP addresses contain a value for each segment, and that hostnames do not contain any invalid characters.
```

Comment:
will the user know what characters are invalid for hostnames? Is it according to general protocols or specific to this config?

Reply:
This prop adheres to the common hostname format defined in RFC 123, and IMO it's pretty well understood what characters are valid for a hostname, so I think it's fine to leave the message as-is